### ISSUE TITLE
[1.0.0] Refactor "part stock history" into plugin

### ISSUE THREAD

**Issue by SchrodingersGat** [inventree/InvenTree#8671]
The "stocktake" part panel can be moved to a plugin, to allow for greater customization and removing a complex piece of code which some users may not want anyway.

**Comment by matmair** [comment]
@SchrodingersGat what specific function are you referring to here? The ability to do a stock take or stock change tracking in general?

**Comment by matmair** [comment]
@SchrodingersGat you still see this as a 1.0 feature?

**Comment by SchrodingersGat** [comment]
Yes, currently working on this. The [inventree-stock-forecasting](https://github.com/inventree/inventree-stock-forecasting) plugin is done, the "stock history" is a bit more complex

**Comment by SchrodingersGat** [comment]
Closed in https://github.com/inventree/InvenTree/pull/10124
